draft-ietf-quic-transport.md

```diff
@@ -2696,11 +2741,12 @@ controller.
 
 ### Response to a RST_STREAM
 
-Since streams are bidirectional, a sender of a RST_STREAM needs to know how many
-bytes the peer has sent on the stream.  If an endpoint receives a RST_STREAM
-frame and has sent neither a FIN nor a RST_STREAM, it MUST send a RST_STREAM in
-response, bearing the offset of the last byte sent on this stream as the final
-offset.
+RST_STREAM terminates a stream abruptly.  Whether any action or response can or
```

Comment:
This wording is a bit confusing now.  I think this should be "RST_STREAM closes the sender's direction of a stream abruptly." or something similar.  Also, this isn't just a new frame being added, this is essentially saying only the sender can close the send side of a stream.